dictionary/homework_dictionary.py

```diff
@@ -0,0 +1,55 @@
+"""Solution task 1"""
+check_name = input("Please, enter your name:  ").title()
+roles_dict = {
+    'admin': ['George', 'Ruslan', 'Valeriy', 'Irina'],
+    'maintainer': ['Julia', 'Alex'],
+    'manager': ['Piter', 'Max', 'Sophia'],
+    'developer': ['Natali', 'Nikita', 'Diana']
+}
+list_system_names = []
+system_role = ''
+
+for roles, list_names in roles_dict.items():
+    for names in list_names:
+        list_system_names.append(names)
+        if check_name in names:
+            system_role = roles
+        else:
+            pass
+"""I think for correcting end current loops should be block else,
+but I don`t know what to write in this block else.
+Is it correct if i write else: pass?"""
+
+if check_name in list_system_names:
+    print(f"Hello {system_role}")
+else:
+    print("Hello guest")
+
+"""Solution task 2"""
+params = {'v': 'some_id', 'list': 'some_list', 'index': '6', 't': 'Os'}
+initial_str = 'https://www.youtube.com/watch?'
+
+params_list = []
+for key, value in params.items():
+    params_list.append(key + '=' + value)
+
+params_string = '&'.join(params_list)
+print(f"Result link is {initial_str + params_string}")
```

Comment:
Can be better
![image](https://github.com/Pticatos86/test_repos_python/assets/51053483/6f0a91d1-5c51-4db2-a64a-2122b7e1f118)

Reply:
fixed